app/conf/Config.scala

```diff
@@ -49,19 +50,24 @@ object Config {
     val clientSecret = requiredString(config, "auth.google.clientSecret")
     val domain = requiredString(config, "auth.domain")
     val redirectUrl = s"${requiredString(config, "host")}/oauthCallback"
+
+    @nowarn
```

Comment:
These `@nowarn` calls are required because we have enabled `-Xfatal-warnings` for this build, in line with Scala Best Practices. Deprecation is a warning, but with `-Xfatal-warnings` enabled this is a compile error. Luckily, Scala 2.13 introduces `@nowarn` so we can temporarily ignore these warnings during this multi-step upgrade process.